lang/src/org/partiql/lang/eval/NaturalExprValueComparators.kt

```diff
@@ -126,11 +139,15 @@ enum class NaturalExprValueComparators(private val nullOrder: NullOrder) : Compa
     ): Int {
         val pairCmp = object : Comparator<Pair<T, Int>> {
             override fun compare(o1: Pair<T, Int>, o2: Pair<T, Int>): Int {
-                val cmp = entityCmp.compare(o1.first, o2.first)
+                val (leftPair, rightPair) = when(order) {
```

Comment:
From the ktlint error:

```suggestion
                val (leftPair, rightPair) = when (order) {
```